appveyor/README.md

```diff
@@ -44,3 +16,18 @@ optimize downloads and caching for Appveyor.com CI builds.
     * protoc 2.5.1
     * Boost 1.67
     * OpenSSL 1.0.2j (64-bit) from http://slproweb.com/products/Win32OpenSSL.html
+* Version 17.05 splits the single zip file into several to stay under Github's
+  file size limits.
+  * Built using and for Visual Studio 2017 with prebuilt OpenSSL
+  * `rippled_deps.protoc.2.5.1.zip`
+    * Includes:
+      * protoc 2.5.1
+  * `rippled_deps.openssl.1.0.2j.zip`
+    * Includes:
+      * OpenSSL 1.0.2j (64-bit) from http://slproweb.com/products/Win32OpenSSL.html
+  * `rippled_deps.boost.1.70.zip`
+    * Includes:
+      * Boost 1.70, except for the built library binaries in `stage`
+  * `rippled_deps.boost.stage.1.70.zip
+    * Includes:
+      * Boost 1.70, only the built library binaries in `stage`
```

Comment:
Previous archives used the directory name `stage64` vs `stage`. Likely doesn't matter or an easy path fix but just thought I'd mention it.

Reply:
It doesn't seem to matter. Appveyor works fine either way. If you think it's important enough, I'll rename.